### PR TITLE
Correct spelling

### PR DESCRIPTION
**Pull request by sdebruijn**
**Description**

- Correction of a few spelling mistakes.
- Capitalize a Grafana dashboard parameter like other parameters

### CODE REVIEW

File: custom_dashboard/grafana_dashboards/tagging-compliance-dashboard.json
Comment (yaseenzafar123) [inline custom_dashboard/grafana_dashboards/tagging-compliance-dashboard.json:1]: @sdebruijn This is not required

File: infrastructure/terraform.test.tfvars [hunk 1]
```diff
@@ -20,17 +20,13 @@ public_subnet_cidr_block = {
   "eu-west-1a" = "15.0.1.0/24"
   "eu-west-1c" = "15.0.2.0/24"
 }
-<<<<<<< HEAD
```
Comment (samanXgrid) [inline infrastructure/terraform.test.tfvars:23]: @sdebruijn can you please take a pull in your branch as I see this issue is not exist anyone

File: infrastructure/modules/xc3/variables.tf [hunk 2]
```diff
@@ -77,10 +77,7 @@ variable "project" {
 variable "domain_name" {
   type        = string
   description = "Domain name for Grafana Dashboard"
-<<<<<<< HEAD
```
Comment (samanXgrid) [inline infrastructure/modules/xc3/variables.tf:80]: same here I see this issue is fixed in develop and main branch

File: custom_dashboard/grafana_dashboards/tagging-compliance-dashboard.json [hunk 3]
```diff
@@ -192,7 +192,7 @@
           "hide": 0,
           "includeAll": false,
           "multi": true,
-          "name": "region",
+          "name": "Region",
```
Comment (yaseenzafar123) [inline custom_dashboard/grafana_dashboards/tagging-compliance-dashboard.json:195]: @sdebruijn This is not required.

Comment (sdebruijn) [reply custom_dashboard/grafana_dashboards/tagging-compliance-dashboard.json:195]: Account, Project and Month all start with a capital letter. I thought I give 'region' the same treatment. 

I see that the same is true for the home-dashboard and iam-user-dashboard.

Happy to change it back if required.

File: infrastructure/terraform.test.tfvars (hunk 1 above)
Comment (sdebruijn) [reply infrastructure/terraform.test.tfvars:23]: I'm not quite used to this process. Do I need to pull from XgridInc/xc3:main? I think my editor shows it is up-to-date with that branch. Am I doing something wrong?

File: custom_dashboard/grafana_dashboards/tagging-compliance-dashboard.json (hunk 3 above)
Comment (yaseenzafar123) [reply custom_dashboard/grafana_dashboards/tagging-compliance-dashboard.json:195]: Yeah you can change it back. Thank you.

Comment (sdebruijn) [reply custom_dashboard/grafana_dashboards/tagging-compliance-dashboard.json]: Changed back.

File: infrastructure/terraform.test.tfvars (hunk 1 above)
Comment (sdebruijn) [reply infrastructure/terraform.test.tfvars]: I think it is resolved

**Review by yaseenzafar123 — APPROVED** [review]
LGTM

File: README.md [hunk 4]
```diff
@@ -145,7 +145,7 @@ case 2: user not able to change/update/modify default dashboards in Grafana UI
 
 <br clear="all">
 
-## Contibutor
+## Contributing
```
Comment (muhdsaad) [inline README.md:148]:
```suggestion
## Contributor Guide
```

**Review by muhdsaad — COMMENTED** [review]
LGTM
Kindly look into 1 small suggestion

**Review by muhdsaad — APPROVED** [review]
LGTM

**Review by samanXgrid — APPROVED** [review]
LGTM

**Pull request merged**